### PR TITLE
fix plugins blog and survey

### PR DESCRIPTION
**Pull request by paddeo**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jpsurvey/src/main/java/com/agiletec/plugins/jpsurvey/apsadmin/survey/package_it.properties [hunk 1]
```diff
@@ -114,15 +131,16 @@ jpsurvey_answers=Risposte
 jpsurvey_voters=Votanti
 
 jpsurvey_freeText_results=Risposte a testo libero
-jpsurvey_freeText_results_summary=Nella tabella vengono mostrate le risposte date come testo libero nel questionario. Inoltre sono mostrate le occorrenze di ogni risposta. 
+jpsurvey_freeText_results.help=TO be inserted
```
Comment (nicpuddu) [inline plugins/entando-plugin-jpsurvey/src/main/java/com/agiletec/plugins/jpsurvey/apsadmin/survey/package_it.properties]: missing label?

File: plugins/entando-plugin-jpsurvey/src/main/java/com/agiletec/plugins/jpsurvey/apsadmin/survey/package_it.properties [hunk 2]
```diff
@@ -6,40 +6,57 @@ format.roundedPercentage = {0,number,#}
...
 title.jpsurvey.trash.survey=Cancella Questionario
+title.jpsurvey.trash.survey.help=Il componente SURVEY consente all'amministratore di creare e gestire Sondaggi e Questionari. Dalle pagine di configurazione gli amministratori possono accedere alle liste dei sondaggi e dei questionari, modificare o creare nuovi sondaggi e questionari, decidere se pubblicarli e accedere ai risultati.
 title.jpsurvey.trash.poll=Cancella Sondaggio
+title.jpsurvey.trash.poll.help=Il componente SURVEY consente all'amministratore di creare e gestire Sondaggi e Questionari. Dalle pagine di configurazione gli amministratori possono accedere alle liste dei sondaggi e dei questionari, modificare o creare nuovi sondaggi e questionari, decidere se pubblicarli e accedere ai risultati.
 title.jpsurvey.poll.edit.intro=In questa sezione puoi compilare il sondaggio creando Domande e Risposte. Puoi inoltre associare una immagine al Sondaggio.
 title.jpsurvey.survey.edit.intro=In questa sezione puoi compilare il questionario creando Domande e Risposte. Puoi inoltre associare una immagine al Questionario.
 title.jpsurvey.survey.results=Risultati
+title.jpsurvey.survey.results.help=TO be inserted
```
Comment (nicpuddu) [inline plugins/entando-plugin-jpsurvey/src/main/java/com/agiletec/plugins/jpsurvey/apsadmin/survey/package_it.properties]: missing label?

Comment (paddeo) [inline plugins/entando-plugin-jpsurvey/src/main/java/com/agiletec/plugins/jpsurvey/apsadmin/survey/package_it.properties]: yes

File: plugins/entando-plugin-jpsurvey/src/main/java/com/agiletec/plugins/jpsurvey/apsadmin/survey/package_it.properties (hunk 1 above)
Comment (paddeo) [inline plugins/entando-plugin-jpsurvey/src/main/java/com/agiletec/plugins/jpsurvey/apsadmin/survey/package_it.properties]: yes

**Comment by entando-service** [comment]
All is Well

**Pull request merged**